### PR TITLE
Implement model manipulation tool (jubamodel) (fix #4)

### LINKED ISSUE #4: Integrate model file manipulation tools
We often need to directly work with saved model files (call `save` RPC, move the file from `/tmp`, modify the model file, copy over to another server, call `load` RPC, etc.)
It is very convenient if these features are built-in to one `jubakit` package.

We already have a CLI tools to do this (but only work with Py2.)

https://github.com/kmaehashi/jubatus-utils

I'd like to integrate `jubamodel` (that can manually modify the model contents and fix corrupt CRC headers) and `jubafetch` (that can seamlessly `save` / `load` model files) commands into jubakit.

### PR DESCRIPTION
**Pull request by kmaehashi**
Implements #4.

This PR adds ``jubamodel`` command and its underlying Python APIs accessible via jubakit.

### CODE REVIEW

File: doc/source/guide/jubamodel.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+.. option:: -C <output_config>, --output-config <output_config>
+
+   Path to the output config file.
+   When specified, Jubatus configuration (JSON) will be extracted from the model and written to the specified file.
+
+.. option:: -f, --no-validate
+
+   When loading model files in ``binary`` format, ``jubamodel`` validates the model data structure (including CRC32 checksum).
+   When this option is specified, these validation will be disabled.
```
Comment (rimms) [inline doc/source/guide/jubamodel.rst]: s/these validation/these validations/ ?

**Comment by rimms** [comment]
In Python 3.5.1, I faced following error:

```bash
$ jubamodel ~/shogun.jubatus -o json
{
  "user_raw": " kgGSkpLeACq4bmFtZST....
   .. snip ..
    "type": "classifier"
  },
  "user": {
    "user_data": [
      [
        [
          {
            Error: None: failed to write model (TypeError): key b'name$\xe6\x99\xb4@unigram#bin/bin' is not a string
```

In Python 2.7.9, this error does not occur.

I used model of [shogun example](https://github.com/jubatus/jubatus-example/tree/master/shogun). This model contains unicode data.

File: doc/source/guide/jubamodel.rst (hunk 1 above)
Comment (kmaehashi) [inline doc/source/guide/jubamodel.rst]: Fixed

**Comment by kmaehashi** [comment]
>             Error: None: failed to write model (TypeError): key b'name$\xe6\x99\xb4@unigram#bin/bin' is not a string

I fixed so that jubamodel assume all strings are encoded in UTF-8.

**Comment by rimms** [comment]
👍 LGTM

**Pull request merged**